migrations/0000_profile_functions.sql

```diff
@@ -0,0 +1,46 @@
+CREATE or replace function create_profile(varchar(32), varchar(256), varchar(256), timestamp, timestamp, varchar(128)) returns void as
+$$
+begin
+    insert into profile (username,
+                         password,
+                         avatar_link,
+                         last_activity,
+                         created_at,
+                         created_from_ip,
+                         deleted_at,
+                         is_active)
+    values ($1, $2, $3, $4, $5, $6, null, true);
+end
+$$ language 'plpgsql';
+
+CREATE or replace function get_profile(varchar(32)) returns setof profile as
+$$
+begin
+    return query select * from profile where username = $1;
+end
+$$
+    language 'plpgsql';
+
+CREATE or replace function update_profile(varchar(32), varchar(256), varchar(256), timestamp,
```

Comment:
Use next function declaration:

CREATE OR REPLACE FUNCTION update_profile(password profile.password%TYPE ... ) returns ...

If you change the type of a table column, you will not need to change the functions that depend on that column.